application/sitebuilder/build.py

```diff
@@ -62,6 +75,35 @@ def enforce_static_mode():
             clear_up(build_dir)
 
 
+def build_and_upload_error_pages(application):
```

Comment:
Maybe a comment here about why error pages are build and pused separately from the rest of the site?  BUt maybe it's obvious enough :man_shrugging: